app/components/Button/Button.css

```diff
@@ -57,38 +73,118 @@ html[data-theme='dark'] .button {
   margin-left: 8px;
 }
 
-.dark {
-  background: var(--color-red-3);
+.button.dark {
+  background: var(--color-red-2);
+  border: 1px solid rgba(--color-gray-4, 20%);
   color: var(--color-white);
-  border-color: var(--color-red-1);
+  box-shadow: none;
 
   &.fa {
     color: var(--color-white);
   }
 
-  &:active {
+  &:focus-visible {
+    box-shadow: 0 0 0 3px rgba(--color-red-3, 25%);
+  }
+
+  &:hover:not(:disabled) {
+    background: var(--color-red-1);
+    border: 1px solid rgba(--color-gray-5, 20%);
+  }
+}
+
+html[data-theme='dark'] .button.dark {
+  background: var(--color-red-1);
+  border: 1px solid rgba(--color-light-gray-4, 20%);
+  color: var(--color-black);
+  box-shadow: none;
+
+  &:focus-visible {
+    box-shadow: 0 0 0 3px rgba(--color-red-5, 25%);
+  }
+
+  &:hover:not(:disabled) {
+    background: var(--color-red-2);
+    border: 1px solid rgba(--color-light-gray-5, 20%);
+  }
+}
```

Comment:
Most of these are still the same as the `.button.dark` above. Pls remove all the duplicate stuff, and it would be way better if you could just modify the dark-theme color variables to be proper, or add new ones, instead of having custom dark-theme color adjustments here.

Reply:
Removed all duplicate code 👍🏼